packages/apps/cdn-viewer/package.json

```diff
@@ -21,7 +21,7 @@
     "@bentley/presentation-frontend": "^2.14.0",
     "@bentley/product-settings-client": "^2.14.0",
     "@bentley/rbac-client": "^2.14.0",
-    "@bentley/react-scripts": "3.4.1",
+    "@bentley/react-scripts": "3.4.9",
```

Comment:
i would actually bump this to 3.4.11 throughout the monorepo

Reply:
done